Resources/public/js/views/fields/ez-relation-editview.js

```diff
@@ -0,0 +1,191 @@
+    "use strict";
+    /**
+     * Provides the field edit view for the relation fields
+     *
+     * @module ez-relation-editview
+     */
+    Y.namespace('eZ');
+
+    var FIELDTYPE_IDENTIFIER = 'ezobjectrelation';
+
+    /**
+     * Relation edit view
+     *
+     * @namespace eZ
+     * @class RelationEditView
+     * @constructor
+     * @extends eZ.FieldEditView
+     */
+    Y.eZ.RelationEditView = Y.Base.create('relationEditView', Y.eZ.FieldEditView, [Y.eZ.AsynchronousView], {
+
+        events: {
+            '.ez-relationeditview-remove': {
+                'tap': '_removeRelation',
+            },
+            '.ez-discover': {
+                'tap': '_runUniversalDiscoverySettings',
+            },
+        },
+
+        initializer: function () {
+            this._fireMethod = this._fireLoadFieldRelatedContent;
+            this._watchAttribute = 'destinationContent';
+            if( this.get('field').fieldValue.destinationContentId ){
+                this._set('destinationContentId', this.get('field').fieldValue.destinationContentId);
+            }
+        },
+
+        /**
+         * Fire the `loadFieldRelatedContent` event
+         *
+         * @method _fireLoadFieldRelatedContent
+         * @protected
+         */
+        _fireLoadFieldRelatedContent: function () {
+            if ( !this._isFieldEmpty() ) {
+                this.fire('loadFieldRelatedContent', {
+                    fieldDefinitionIdentifier: this.get('fieldDefinition').identifier
+                });
+            }
+        },
+
+        /**
+         * Checks whether the field is empty
+         *
+         * @method _isFieldEmpty
+         * @protected
+         * @return {Boolean}
+         */
+        _isFieldEmpty: function () {
+            return !this.get('destinationContentId');
+        },
+
+        /**
+         * Returns an object containing the additional variables
+         *
+         * @method _variables
+         * @protected
+         * @return Object
+         */
+        _variables: function () {
+            var dest = this.get('destinationContent');
+
+            return {
+                destinationContent: dest ? dest.toJSON() : null,
+                loadingError: this.get('loadingError'),
+                isEmpty: this._isFieldEmpty(),
+                isRequired: this.get('fieldDefinition').isRequired,
+            };
+        },
+
+        /**
+         * Tap event handler for the remove button. It resets the relation.
+         *
+         * @method _removeRelation
+         * @protected
+         * @param {Object} e
+         */
+        _removeRelation: function (e) {
+            e.preventDefault();
+            this._set('destinationContentId', null);
+            this.set('destinationContent', null);
+            this.validate();
+        },
+
+        /**
+         * Check if required and set the error status
+         *
+         * @method validate
+         * @protected
```

Comment:
not protected anymore... But since you are redefining a method of the parent class, you don't need to add a comment.